src/core/Akka.Remote.Tests/RemotingTerminatorSpecs.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using Akka.Actor;
+using Akka.Configuration;
+using Akka.TestKit;
+using Xunit;
+
+namespace Akka.Remote.Tests
+{
+    /// <summary>
+    /// Validate that the remoting terminator for Akka.Remote transports fires and does not
+    /// indefinitely block the <see cref="ActorSystem"/> from terminating.
+    /// </summary>
+    public class RemotingTerminatorSpecs : AkkaSpec
```

Comment:
Used to verify that ActorSystems can shutdown correctly when using the `RemotingTerminator`